### DISCUSSION TITLE
How can i install the latest buildah available with any package manager?

### DISCUSSION THREAD

**Discussion by yaronkaikov** [containers/buildah#5705] [General]
Hi
I am trying to install the latest buildah, working with ubuntu 24.04 and Fedora 37,

Other then installation from source, is there a way to use `dnf/apt-get` to install the latest ?

**Comment by edsantiago** [comment]
Fedora 37 went EOL last year. Have you considered upgrading?

**Reply by yaronkaikov** [reply]
yes, i see that fedora40 has the latest releases. What about ubuntu 24.04?

**Reply by rhatdan** [reply]
We do not keep track of versions on other distributions then Fedora, RHEL and Centos Stream.

Some contributors have built repositories for updates of Ubuntu for Podman but I have not heard of any for Buildah.
